insights/parsers/systemd/unitfiles.py

```diff
@@ -45,6 +48,19 @@ def is_on(self, service_name):
             service_name (str): service name including '.service'
 
         Returns:
-            bool: True if service is enabled, False otherwise
+            Union[bool, None]: True if service is enabled, False if it is disabled. None if the
+                               service doesn't exist.
         """
-        return self.services.get(service_name, False)
+        return self.services.get(service_name, None)
```

Comment:
I'm a little wary of tri-state booleans, but I'm happy enough since code that does 'if obj.is_on(service)' will still work correctly.  Nothing should be checking the exact return value for False.